lib/ansible/modules/cloud/ovirt/ovirt_templates.py

```diff
@@ -154,6 +173,42 @@
   cluster: mycluster
   id: 1111-1111-1111-1111
 
+# Register template with vnic profile mappings
+- ovirt_templates:
+  state: registered
+  storage_domain: mystorage
+  cluster: mycluster
+  id: 1111-1111-1111-1111
+  vnic_profile_mappings:
+    - source_network_name: mynetwork
+      source_profile_name: mynetwork
+      target_profile_name: target_network
+    - source_network_name: mynetwork2
+      source_profile_name: mynetwork2
+      target_profile_name: target_network2
+
+# Register template with mapping
+- ovirt_templates:
```

Comment:
Just thinking out loud.

Maybe it would be nicer to have:

```yaml
mappings:
  domain:
    - source_name: domainA
      destination_name: domainB
    - source_name: domainXA
      destination_name: domainXB
  role:
    - source_name: roleA
      destination_name: roleB
  cluster:
    - source_name: roleA
      destination_name: roleB
```

Since we just set it, and don't update those params it's not really needed to have flat structure imho. but the way it's done for now is nice as well.

Reply:
You got a point there.
The thing is that those parameters in the REST are configured as types (Role, Cluster, Domain, etc.)
It could be a great benefit, in the future, if we will want to map extra fields in the destination for example, and that way we will use a type instead of prefix.